ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/internal/response/JsonRpcError.java

```diff
@@ -101,7 +101,24 @@
   CANT_CONNECT_TO_LOCAL_PEER(-32100, "Cannot add local node as peer."),
 
   // Invalid input errors
-  ENODE_ID_INVALID(-32000, EnodeURL.INVALID_NODE_ID_LENGTH);
+  ENODE_ID_INVALID(-32000, EnodeURL.INVALID_NODE_ID_LENGTH),
+
+  // Enclave errors
+  NODE_MISSING_PEER_URL(-50200, "NodeMissingPeerUrl"),
```

Comment:
Question:  Is there a reason we use the same error code for every error?  Would it make it easier to debug if everything used unique codes?

Reply:
There is no documentation regarding what the error code for Enclave related errors should be in the JSON-RPC spec for ethereum clients.